bpfd/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use crate::command::{
 
 pub async fn serve(config: Config, static_programs: Vec<StaticPrograms>) -> anyhow::Result<()> {
     let (tx, mut rx) = mpsc::channel(32);
-    let addr = "[::1]:50051".parse().unwrap();
+    let addr = config.grpc.endpoint.address.parse().unwrap();
```

Comment:
I get that this `unwrap()` existed previously, but it also operated on an unchanging literal string before (and as such, would never fail). Now that we're taking input for the address parsing, we should remove the unwrap and handle the result here.

Reply:
This makes sense, my only question would be how do we want to handle this exactly. Do we want to fallback to the default address or stop execution? In case we'd like the latter, do we want to `expect` with a meaningful message or return the error via `?` (I'd assume we want this last one).

Reply:
@astoycos should probably chime in here but my vote would be to fail fast when user provided configuration is bad, and return the error here such as to force the user operating BPFD to fix the configuration before it will continue work.

Reply:
I think we should stop execution and fail with a meaning error.

Reply:
Changed `parse` to `expect`, hopefully that is enough.

Reply:
Clippy suggested using `unwrap_or_else(|_| panic!(...))`, I assume this is because with this approach the string formatting is only evaluated if the unwrap fails.

Reply:
Yep sounds good!